src/containers/Event/Settings/Settings.css

```
@@ -1,5 +1,12 @@
 .Settings-container {
-    display: flex;
-    flex-direction: column;
-    padding: 10px;
+  padding: 10px;
+}
+
+.Edit-container {
```

Use the component name as prefix because CSS class are global. 

```suggestion
.Settings-container {
```